doc/QuickStart.md

```diff
@@ -19,6 +19,14 @@
 
 [↑ Go to the Table of Content ↑](../README.md) | [Continue to Modules Development →](ModulesDevelopment.md)
 
+## Read this first !
+
+This page will give you a basic idea of the QC and how to run it. Please read it *in its entirety* and run the commands along the way. Do not start developing your module before you have reached the next section called "Modules Development". 
```

Comment:
Oh, one idea here. If you agree with me, can you indicate that one should pull the latest QC version if they read the latest documentation?

Reply:
good idea!